### PR TITLE
Add idiot

### PR DESCRIPTION
**Pull request by codingneko**
Really wanted to add a CWed subject to test out the feature in production, I think this one is mild enough, hope you agree :3

### CODE REVIEW

File: arrays.ts [hunk 1]
```diff
@@ -298,4 +298,5 @@ export const SUBJECTS: Entry[] = [
   "guv'nor",
   'captain',
   'princess',
+  { word: 'bottom', cw: 'kinky' },
```
Comment (olivvybee) [inline arrays.ts]: I guess it nicely avoids the potential for `cw: lewd, lewd` but I don't really see `bottom` as "kinky" per se 🤔

Comment (codingneko) [reply arrays.ts]: uh fair... perhaps D/s would be better? Though tbh it's not really related... Bottom and sub are two different things xd

**Comment by codingneko** [comment]
updated with a different word that may be more fitting

**Comment by codingneko** [comment]
omfg I borked this pull request, sorry ahhaha I'm going to create a new one xd

**Pull request closed without merging**